docker/clickhouse/server/config.d/macros.xml

```diff
@@ -0,0 +1,7 @@
+<clickhouse>
+    <macros>
+        <cluster>$CLUSTER</cluster>
```

Comment:
I don't think the cluster needs to be in a macro. It can just be hard-coded in the zookeeper path.